_includes/header.html

```diff
@@ -1,217 +1,181 @@
-          <div class="syn-dropdown-list">
-            <div class="syn-dropdown-list-item syn-body-2 syn-mb-0">
-              <a href="/resources">Resources</a>
-            </div>
-            <div class="syn-dropdown-list-item syn-body-2 syn-mb-0">
-              <a href="{{ site.baseurl }}">Docs</a>
-            </div>
-            <div class="syn-dropdown-list-item syn-body-2 syn-mb-0">
-              <a href="https://algorithmia.com/blog" target="_blank"
-                >Blog
-                <svg class="syn-ml-4" width="14" height="14" viewBox="0 2 24 20">
-                  <path
-                    fill="currentColor"
-                    d="M19 19H5V5h7V3H5c-1.11 0-2 .9-2 2v14c0 1.1.89 2 2 2h14c1.1 0 2-.9 2-2v-7h-2v7zM14 3v2h3.59l-9.83 9.83 1.41 1.41L19 6.41V10h2V3h-7z"
-                  />
-                </svg>
-              </a>
-            </div>
-            <div class="syn-dropdown-list-item syn-body-2 syn-mb-0">
-              <a href="https://learn.algorithmia.com" target="_blank"
-                >Learning Center
-                <svg class="syn-ml-4" width="14" height="14" viewBox="0 2 24 20">
-                  <path
-                    fill="currentColor"
-                    d="M19 19H5V5h7V3H5c-1.11 0-2 .9-2 2v14c0 1.1.89 2 2 2h14c1.1 0 2-.9 2-2v-7h-2v7zM14 3v2h3.59l-9.83 9.83 1.41 1.41L19 6.41V10h2V3h-7z"
-                  />
-                </svg>
-              </a>
-            </div>
-          </div>
+      </div>
+    </div>
+  </div>
+  <div onclick="['nav-open'].forEach((a) => document.querySelector('.syn-page-header').classList.toggle(a))"></div>
+  <div class="syn-page-header-content syn-flex justify-space-between">
+    <div class="syn-flex align-center">
+      <a class="syn-flex" href="https://algorithmia.com">
+        <img class="syn-ml-8" src="https://algorithmia.com/v1/config/logo" style="max-height: '24px'; max-width: '100%';" />
+      </a>
+    </div>
+    <button class="syn-btn icon text grow syn-hidden-lg" onclick="['nav-open'].forEach((a) => document.querySelector('.syn-page-header').classList.toggle(a))">
+      <svg width="32" height="20" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" class="open-button"><g stroke="currentColor" stroke-width="1.5" fill="none" fill-rule="evenodd" stroke-linecap="round"><path d="M1 1h29.849M1 10h29.849M1 19h29.849"></path></g></svg>
+      <svg width="24" height="24" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" class="close-button"><g stroke="currentColor" stroke-width="1.5" fill="none" fill-rule="evenodd" stroke-linecap="round"><path d="M1.371 22.553L22.478 1.447M1.371 1.461l21.107 21.107"></path></g></svg>
+    </button>
+    <div class="syn-visible-lg">
+      <a href="https://algorithmia.com/mlops" class="syn-link-no-decoration">
+        <button class="syn-btn text">MLOps</button>
+      </a> 
+      <a href="https://algorithmia.com/product" class="syn-link-no-decoration">
```

Comment:
Reason for all of these updated hrefs is because some customer's are running inside of a no internet environment. In that case we want the dev-center to be rendered via the container within the customer's cluster, not the public marketplace one.
```suggestion
      <a href="/product" class="syn-link-no-decoration">
```

Reply:
Actually, disregard the above comment. These are marketing pages which won't exist within an enterprise cluster. It's fine to leave them as you have them since enterprise customers won't be clicking on marketing routes anyway.